src/index.html

```diff
@@ -129,12 +146,54 @@ <h2 class="modal__title">New Post</h2>
     </div>
     <div class="modal" id="preview-modal">
       <div class="modal__content modal__content_type_preview">
-        <button aria-label="close modal" type="button" class="modal__close-button modal__close-button_type_preview"></button>
-        <img class="modal__image" src="./images/2-photo-by-ceiline-from-pexels.jpg" alt="image"/>
+        <button
+          aria-label="close modal"
+          type="button"
+          class="modal__close-button modal__close-button_type_preview"
+        ></button>
+        <img class="modal__image" src="#" alt="image" />
         <p class="modal__caption"></p>
       </div>
     </div>
-    <script src="./scripts/validation.js"></script>
-    <script src="./scripts/index.js"></script>
+    <div class="modal" id="avatar-modal">
+      <div class="modal__container">
+        <button type="button" class="modal__close-button"></button>
+        <h2 class="modal__title">Edit Profile Avatar</h2>
+        <form class="modal__form-avatar" id="edit-avatar-form">
+          <label for="profile-avatar-input" class="modal__label">
+            Profile Picture
+            <input
+              id="profile-avatar-input"
+              type="url"
+              class="modal__input"
+              name="avatar"
+              placeholder="Paste a link to an image"
+              required
+            />
+            <span id="profile-avatar-input-error" class="modal__error">
+              Please enter a URL
+            </span>
+          </label>
+          <button type="submit" class="modal__submit-button">Save</button>
+        </form>
+      </div>
+    </div>
+    <div class="modal" id="delete-modal">
+      <div class="modal__delete-container">
+        <button
+          type="button"
+          class="modal__close-button modal__close-button_type-delete"></button>
+        <h2 class="modal__title-delete">
+          Are you sure you want to delete this image?
+        </h2>
+        <form class="modal__form-delete" id="delete-form">
+          <div class="modal__delete-content">
+            <div class="modal__delete-btn">
+              <button type="submit" class="modal__delete-button">Delete</button>
+              <button type="button" class="modal__cancel-button">Cancel</button>
+          </div>
```

Comment:
One more `</div>` is needed here to close the `div` from line 190


```html
        <form class="modal__form-delete" id="delete-form">
          <div class="modal__delete-content">
            <div class="modal__delete-btn">
              <button type="submit" class="modal__delete-button">Delete</button>
              <button type="button" class="modal__cancel-button">Cancel</button>
            </div>
          </div>
        </form>
      </div>
    </div>
  </body>
</html>

```

Reply:
* It's necessary to fix all the errors (only **errors**, not warnings) found by html `validator`: https://validator.w3.org/nu/#textarea . Just copy your HTML code to the text area and hit `Check`. This validator will help you find and fix small typos, syntax errors and other similar issues. This short **video** shows how to use validators for your code: https://youtu.be/Yc_Qw4-oVq0